packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -196,15 +198,28 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
     if (this.state.isLegendHovered || this.state.isLegendSelected) {
       shouldHighlight = this.state.selectedLegendTitle === lineLegendText;
     }
+
     const line = (
-      <path
-        opacity={shouldHighlight ? 1 : 0.4}
-        d={linePath(lineData)!}
-        fill={'none'}
-        strokeWidth={3}
-        stroke={lineLegendColor}
-      />
+      <>
+        <path
+          opacity={shouldHighlight ? 1 : 0.4}
+          d={linePath(lineData)!}
+          fill="transparent"
+          strokeLinecap="square"
+          strokeWidth={3 + LINE_BORDER_WIDTH}
```

Comment:
make this a prop. Dont make it mandatory.

Reply:
ok, i will make this a prop.